client/.babelrc

```diff
@@ -1,11 +1,13 @@
 {
   "presets": [
```

Comment:
could you change the commit line to be "graphql: add support for enum types"? that's more like the go style guide.